bpf/lib/static_data.h

```diff
@@ -26,4 +26,6 @@ DEFINE_U32_I(NAME, 4) = bpf_htonl((a13) << 24 | (a14) << 16 | (a15) << 8 | (a16)
 DEFINE_U32_I(NAME, 1) = (a1) << 24 | (a2) << 16 |  (a3) << 8 | (a4);	\
 DEFINE_U32_I(NAME, 2) =                            (a5) << 8 | (a6)
 
+#define DEFINE_U32_V(NAME, value) volatile __u32 NAME = value
```

Comment:
Is the use of `volatile` required here?

Reply:
The goal of the `volatile` here is to ensure that the compiler does not make any assumptions about the value held by `NAME`, so that at compile time both paths are generated, then at load time we can substitute the appropriate value.

I expect that this is particularly important when the value of `NAME` is used for conditional code execution.

Reply:
Ah, I see. It seemed weird that above macros didn't have the same, but that makes sense.